code/game/objects/items/stacks/medical.dm

```diff
@@ -42,9 +42,9 @@
 				user << "<span class='warning'>[H] isn't bleeding!</span>"
 				return
 
-	if(isliving(M))
-		if(!M.can_inject(user, 1))
-			return
+	// if(isliving(M))
```

Comment:
It's the same check used for syringes and stuff, yes.
I'm only removing it's proccall in the medical.dm where bruisepacks/ointments are located.